catalog/spark/spark.bom

```diff
@@ -271,6 +283,13 @@ brooklyn.catalog:
           type: integer
           default: 3
         - *sparkDownloadUrl
+        - name: spark.sharedsecuritygroup.create
+          label: "Create Driver UI SharedSecurityGroup"
+          description: |
+            Kubernetes blueprint will configure security groups to allow access to the
+            driver ui port range
```

Comment:
Kubernetes? Why not "Configure cloud security groups to allow instances to communicate on <PORTS>. Supported on clouds: <CLOUDS>", replacing PORTS and CLOUDS as appropriate.